materialize-styles.loader.js

```diff
@@ -7,6 +7,7 @@ module.exports = function(content) {
     var config = this.exec(content, this.resourcePath);
     var start =
             "$roboto-font-path: \"~materialize-css/fonts/roboto/\";\n"
+            + "@import \"~materialize-css/sass/components/_color.scss\";\n";
```

Comment:
If I'm not mistaken, the extra ';' at the end makes the 'start' variable not include "@import \"./materialize.config.scss\";\n", and therefore we won't be able to customize styles through materialize.config.scss.